http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceInterp.scala

```diff
@@ -1126,6 +1110,32 @@ object RuntimeServiceInterp {
       }
     } yield disk
 
+  private def checkAttachAction[F[_]](userInfo: UserInfo,
```

Comment:
Nit, would it make sense to move this with the other checkAction function in SamUtils? Might not be in the scope for this PR, but disks can also be attached to apps and so it could make sense to move this logic out of the RuntimeServiceInterp file